exercises.ipynb

```diff
@@ -98,74 +74,104 @@
    },
    "outputs": [],
    "source": [
-    "bricks = parse_bricks(START_URL)"
+    "bricks = parse_bricks(START_URL)\n",
+    "\n",
+    "assert(bricks[0] == '10251:  Brick Bank')\n",
+    "assert(bricks[9] == '10722:  Snake Showdown')"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 277,
-   "metadata": {
-    "autoscroll": false,
-    "collapsed": false,
-    "ein.hycell": false,
-    "ein.tags": "worksheet-0",
-    "slideshow": {
-     "slide_type": "-"
-    }
-   },
+   "execution_count": 5,
+   "metadata": {},
    "outputs": [],
    "source": [
-    "assert(bricks[0] == '10251:  Brick Bank')\n",
-    "assert(bricks[9] == '10722:  Snake Showdown')"
+    "# Exercise #2: Now write code that gets you all the links from ALL the pages\n",
+    "\n",
+    "def get_titles(soup):    \n",
```

Comment:
Looks great, by it's worth also being familiar with the CSS notation that we went over in class!